lib/active_storage/blob.rb

```diff
@@ -53,6 +53,11 @@ def upload(io)
     self.checksum  = compute_checksum_in_chunks(io)
     self.byte_size = io.size
 
+    if service.nil?
+      raise "No storage service specified for current env (#{Rails.env}). " \
+            "Add config.active_storage.service = :local into your config/environments/#{Rails.env}.rb."
+    end
```

Comment:
You could have ActiveStorage installed and don't use it anywhere, your app will still die on load — quite frustrating. And that will happen often after the ActiveStore is pulled into Rails. But then, `config.active_storage.service` will probably be in the default config. So I suppose you're right.